tests/bpftool_tests/bpftool_tests.cpp

```diff
@@ -104,38 +78,53 @@ TEST_CASE("prog load map_in_map.o", "[!shouldfail][prog][load]")
     REQUIRE(output == "");
     REQUIRE(result == 0);
 
-    output = run_command("netsh ebpf sh prog", &result);
-    REQUIRE(
-        output == "    ID  Pins  Links  Mode       Type           Name\n"
-                  "======  ====  =====  =========  =============  ====================\n"
-                  "196609     1      0  JIT        xdp            lookup\n\n");
+    output = run_command("bpftool prog show", &result);
     REQUIRE(result == 0);
+    std::string id = std::to_string(atoi(output.c_str()));
```

Comment:
I think we should do this to other netsh tests also. I was earlier trying to run the netsh tests for native code, but then was hitting this issue (the ID was changing).